portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,34 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import com.google.gson.Gson;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some content hard coded. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    ArrayList<String> test = new ArrayList<String>();
+    test.add("test str 1");
+    test.add("test str 2");
+    test.add("test str 3");
+    String strList = lstToJson(test);
+    response.setContentType("application/json;");
+    response.getWriter().println(strList);
+  }
+
+    /**
+   * Converts ArrayList<String> into JSON string using the Gson library. 
+   */
+  private String lstToJson(ArrayList<String> lst) {
```

Comment:
Try to not abbreviate words in method names. There usually isn't a reason to unless it's something super obvious like num vs number.